src/services/api/search/dto/search.result.dto.entry.card.ts

```diff
@@ -15,4 +15,30 @@ export abstract class ISearchResultCard
     description: 'The Card that was found.',
   })
   card!: IAspect;
+
+  @Field(() => String, {
+    nullable: false,
+    description: 'The Hub nameID of the Card.',
+  })
+  hub!: string;
```

Comment:
suggest to name the field hubNameID - and do the same for all the fields, otherwise it is confusing...also please look at the description.